cmd/benchmark.go

```diff
@@ -106,10 +105,10 @@ func benchTypeCommandActionFactory(runner benchrunner.BenchRunner) cobraext.Comm
 
 		packageRootPath, found, err := packages.FindPackageRoot()
 		if !found {
-			return errors.New("package root not found")
+			return fmt.Errorf("package root not found")
 		}
 		if err != nil {
-			return errors.Wrap(err, "locating package root failed")
+			return fmt.Errorf("locating package root failed: %s", err)
```

Comment:
This is not right. If a caller uses `errors.Is` or `errors.As` on this error it will always fail since `err` is no longer wrapped in the returned value. The verb should be %w to replace `errors.Wrap(errr, ...`.

I have a set of gofmt rewrites that will do the changes mechanically if that would be helpful.

Reply:
Good catch :+1:

Reply:
> I have a set of gofmt rewrites that will do the changes mechanically if that would be helpful.

happy to see the contribution :+1:

Reply:
The details are in the commit message of elastic/beats@4ab90f9b3339c6f8a5d198ad469305dcc7bbad53.